Gemfile

```diff
@@ -38,7 +38,8 @@ gem 'mail'
 gem 'mime-types', require: 'mime/types/columnar'
 
 gem 'hiredis'
-gem 'redis', require:  ["redis", "redis/connection/hiredis"]
+# https://github.com/redis/redis-rb/pull/591
+gem 'redis', require:  ["redis", "redis/connection/hiredis"], github: 'tgxworld/redis-rb'
```

Comment:
hmmm very very uneasy with this, is the a PR for it? also don't source from github publish a gem instead